### PR TITLE
Moving tuya 0xE000X cluster to INIT and adding TS000X quirk

### LINKED ISSUE #1118: [Device Support Request] Tuya TS0001 relay
**Is your feature request related to a problem? Please describe.**
Just received Tuya TS0001 relay, 
image
image
https://www.aliexpress.com/item/1005002482791975.html

It hasn't quirk, so works just basically.

In Tuya Smart application, it has power outage behavior and external switch type settings. The first setting is already implemented for plugs TS011F and TS0121, so it could implemented without problems. Switch type setting haven't been implemented yet.

Here is a screenshot from Tuya Smart application:
image

**Describe the solution you'd like**
The external switch type attribute could be added.

**Device signature - this can be acquired by removing the device from ZHA and pairing it again from the add devices screen. Be sure to add the entire content of the log panel after pairing the device to a code block below this line.**
```
{
  "node_descriptor": "NodeDescriptor(logical_type=<LogicalType.Router: 1>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.AllocateAddress|RxOnWhenIdle|MainsPowered|FullFunctionDevice: 142>, manufacturer_code=4417, maximum_buffer_size=66, maximum_incoming_transfer_size=66, server_mask=10752, maximum_outgoing_transfer_size=66, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=False, *is_full_function_device=True, *is_mains_powered=True, *is_receiver_on_when_idle=True, *is_router=True, *is_security_capable=False)",
  "endpoints": {
    "1": {
      "profile_id": 260,
      "device_type": "0x010a",
      "in_clusters": [
        "0x0000",
        "0x0003",
        "0x0004",
        "0x0005",
        "0x0006",
        "0xe000",
        "0xe001"
      ],
      "out_clusters": [
        "0x000a",
        "0x0019"
      ]
    }
  },
  "manufacturer": "_TZ3000_ark8nv4y",
  "model": "TS0001",
  "class": "ts011f_plug.Plug_0001"
}

```
**Additional context**
@MattWestb may be you know attribute id for external switch type?

### LINKED ISSUE #1134: [Device Support Request] Aqara Wall Switch EU version (no neutral, double rocker)
**Is your feature request related to a problem? Please describe.**
I have several Aqara Smart Wall Switches (no neutral, single/double rocker) EU versions and they work almost perfectly.

The problem exists only with double rocker version (model WS-EUK02), state of the right rocker is not changed if physically switch it.

**Describe the solution you'd like**
This issue has fixed in [zigbee2mqtt#6524](https://github.com/Koenkk/zigbee2mqtt/issues/6524) and [this](https://github.com/Koenkk/zigbee2mqtt/issues/6524#issuecomment-872421599) looks like a solution.

**Device signature**
```json
{
  "node_descriptor": "NodeDescriptor(logical_type=<LogicalType.EndDevice: 2>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.AllocateAddress: 128>, manufacturer_code=4447, maximum_buffer_size=127, maximum_incoming_transfer_size=100, server_mask=11264, maximum_outgoing_transfer_size=100, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=True, *is_full_function_device=False, *is_mains_powered=False, *is_receiver_on_when_idle=False, *is_router=False, *is_security_capable=False)",
  "endpoints": {
    "1": {
      "profile_id": 260,
      "device_type": "0x0100",
      "in_clusters": [
        "0x0000",
        "0x0002",
        "0x0003",
        "0x0004",
        "0x0005",
        "0x0006",
        "0x0009"
      ],
      "out_clusters": [
        "0x000a",
        "0x0019"
      ]
    },
    "2": {
      "profile_id": 260,
      "device_type": "0x0100",
      "in_clusters": [
        "0x0000",
        "0x0003",
        "0x0004",
        "0x0005",
        "0x0006"
      ],
      "out_clusters": []
    },
    "242": {
      "profile_id": 41440,
      "device_type": "0x0061",
      "in_clusters": [],
      "out_clusters": [
        "0x0021"
      ]
    }
  },
  "manufacturer": "LUMI",
  "model": "lumi.switch.l2aeu1",
  "class": "zigpy.device.Device"
}
```

### PR DESCRIPTION
**Pull request by MattWestb**
Moving the tuya PITA clusters to INIT and adding functions for switch types to one of them.
Adding TS000X device quirks that  is using it and also the TS011F have it but i dont knowing if its being implanted in the devices.

New TS000X devises is testes but not  the last have the user not reported back.

The device type is ON_OFF_LIGHT for all cluster but the user like have it as ON_OFF_PLUG_IN_UNIT then its getting on flash as icon  in ZHA is that  OK ?

The calss class TuyaZBExternalSwitchTypeCluster(CustomCluster): is not having any INIT but looks working OK.
Shall it have more code for working better ?
Then how shall it being implanted ?

Fixes #1134
Fixes #1118

### CODE REVIEW

**Comment by MattWestb** [comment]
Fixes #1134 #1118

File: zhaquirks/tuya/__init__.py [hunk 1]
```diff
@@ -844,6 +846,33 @@ class TuyaZBElectricalMeasurement(CustomCluster, ElectricalMeasurement):
     _CONSTANT_ATTRIBUTES = {AC_CURRENT_MULTIPLIER: 1, AC_CURRENT_DIVISOR: 1000}
 
 
+# Tuya Zigbee Cluster 0xE000 Implementation
+class TuyaZBE000Cluster(CustomCluster):
+    """Tuya manufacturer specific cluster 57344."""
+
+    name = "Tuya Manufacturer Specific"
+    cluster_id = TUYA_CLUSTER_E000_ID
+    ep_attribute = "tuya_is_pita_0"
```
Comment (dmulcahey) [inline zhaquirks/tuya/__init__.py:855]: just curious... what is pita?

Comment (MattWestb) [reply zhaquirks/tuya/__init__.py:855]: Alexei was thinking it was on good name for one unknown tuya cluster but we have find fictions on one of them and implanting it here.
**P**ain
**I**n
**T**he
**A** . . (that you is have in side your pants on the back side)

Comment (dmulcahey) [reply zhaquirks/tuya/__init__.py:855]: LMFAO! OK just wanted to confirm... Totally makes sense now! 😄

**Pull request merged**